lib/src/components/NxSubmitMask/NxSubmitMask.scss

```diff
@@ -7,30 +7,21 @@
 @import '../../scss-shared/_nx-variables';
 @import '../../scss-shared/_nx-container-helpers';
 @import '../../scss-shared/_nx-text-helpers';
+@import '../../scss-shared/_nx-mask-helpers';
 
 .nx-submit-mask {
-  align-items: center;
+  @include background-mask(0);
 
-  background-color: transparentize(#555555, 0.25);
-
-  display: flex;
-  justify-content: center;
   position: absolute;
-
-  top: 0;
-  right: 0;
-  bottom: 0;
-  left: 0;
-
-  z-index: $nx-submit-mask-z-index;
 }
 
 .nx-submit-mask--fullscreen {
+  @include background-mask($nx-submit-mask-z-index);
```

Comment:
Rather than re-invoking the whole mixin this should probably just be an override of `z-index`.

Reply:
> Rather than re-invoking the whole mixin this should probably just be an override of `z-index`.

I actually did it like this to demonstrate changing the `z-index` value in the mixin.

Reply:
> > Rather than re-invoking the whole mixin this should probably just be an override of `z-index`.
> 
> I actually did it like this to demonstrate changing the `z-index` value in the mixin.

Though since the SCSS doesn't appear in the examples that's pointless: https://github.com/sonatype/sonatype-react-shared-components/pull/330/commits/57c71a5b3459fab6b5cb16e417d6671fce40fb2c